aws/cloudformation/cloud_formation_adhoc_standalone.yml.erb

```diff
@@ -52,7 +58,7 @@ Resources:
       ResourceSignal:
         Timeout: PT60M
     Properties:
-      ImageId: <%=adhoc_image_id%>
+      ImageId: <%=image_id%>
```

Comment:
Changed to reflect how our regular pipeline stacks pick an AMI:

https://github.com/code-dot-org/code-dot-org/blob/7404dd2d63be833800298574541ec5eaebc58ec3/aws/cloudformation/cloud_formation_stack.yml.erb#L21-L22